objects/DentateGyrus.hoc

```diff
@@ -134,9 +134,9 @@ begintemplate DentateGyrus
         Arguments: 
             $1 : int :  Global RNG seed
         */
-        n_patterns = 12     // number of patterns to simulate (should be 12 for the full sim)
+        n_patterns = 1     // number of patterns to simulate (should be 12 for the full sim)
```

Comment:
I presume you are going to leave the remainder of code as is and look at pattern separation over the whole 1000ms across different PP input patterns?